run_server.py

```diff
@@ -1,3 +1,4 @@
 from api.server import app
 
-app.run()
+if __name__ == '__main__':
+  app.run(host="0.0.0.0",port=5000)
```

Comment:
5000 is default port for flask if I am not mistaken. So why are we adding this explicitly if not hosting to other port?

Reply:
Done.